chimedb/core/orm.py

```diff
@@ -296,7 +298,22 @@ def connect_database(read_write=False, reconnect=False):
 
     from . import connectdb
 
-    connectdb.connect(reconnect)
+    # Try multiple time to connect
+    for i in range(ntries):
+        try:
+            connectdb.connect(reconnect)
+        except ConnectionError:
+            if i == ntries - 1:
+                logger.warn(f"Failed to connect after {ntries} attempts.")
+                raise
+            else:
+                logger.log(
+                    f"Failed to connect attempt {i+1} of {ntries}. "
+                    f"Trying again in {60} seconds."
+                )
+                time.sleep(60)
```

Comment:
60 seconds seems like a long time, though I don't have any real idea what it should be?!

Reply:
Yeah, it's hard to say since we don't know exactly what the root of the issue is so 60 seconds is just totally arbitrary. We could probably try a much shorter wait period, something like 10 seconds, and go from there. Unfortunately it's tricky to test this